library/src/interactor_impl.cxx

```diff
@@ -231,6 +328,22 @@ class interactor_impl::internals
         self->Window.PrintColoringDescription(log::VerboseLevel::INFO);
         self->Window.PrintSceneDescription(log::VerboseLevel::INFO);
         break;
+      case '1':
```

Comment:
Ok I understand why it's not covered now. Should we keep the enum?  
Maybe just doing:
```cpp
// set front view
self->SetViewOrbit({ 0, 0, 1 }, self);
```
instead of
```cpp
self->SetViewOrbit(ViewType::VT_FRONT, self);
```
would simplify the code and improve the coverage.

Reply:
See discussion on Discord, we can also keep the enum but rmove `VT_BACK` and `VT_LEFT`

Reply:
The idea was that `self->SetViewOrbit(ViewType::VT_FRONT, self);` could state clearly that `VT_FRONT` is being passed as a parameter when the function is called for example.
In my opinion that would be a bit less cryptic than using `{ 0, 0, 1 }` as a parameter when the function is called inside the big switch case.

Reply:
Alright, sounds good. Then, please just remove the unused enum values.